ex7.10_NumPy_sort_array.py

```diff
@@ -0,0 +1,25 @@
+import numpy as np
+
+
+def sort_array_axis(array: np.array, axis: str):
```

Comment:
Use type hints and return types -docum
https://www.geeksforgeeks.org/duck-typing-in-python/
https://stackoverflow.com/questions/125367/dynamic-type-languages-versus-static-type-languages
https://docs.python.org/3/library/typing.html
https://realpython.com/lessons/pros-and-cons-type-hints/